scripts/e2e/cmd/runner/networking-v1-mfu_one_namespace_many_ingresses_test.go

```diff
@@ -172,6 +172,51 @@ var _ = Describe("networking-v1-MFU", func() {
 			Expect(err).To(BeNil())
 		})
 
+		It("[hostname-with-wildcard-priority] request host matchs hostname-extension annotation should work with priority", func() {
+			// create namespace
+			namespaceName = "e2e-hostname-with-wildcard-priority"
+			ns := &v1.Namespace{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: namespaceName,
+				},
+			}
+			klog.Info("Creating namespace: ", namespaceName)
+			_, err = clientset.CoreV1().Namespaces().Create(context.TODO(), ns, metav1.CreateOptions{})
+			Expect(err).To(BeNil())
+
+			// create objects in the yaml
+			path := "testdata/networking-v1/one-namespace-many-ingresses/hostname-with-wildcard-priority/app.yaml"
+			klog.Info("Applying yaml: ", path)
+			err = applyYaml(clientset, crdClient, namespaceName, path)
+			Expect(err).To(BeNil())
+
+			time.Sleep(30 * time.Second)
+
+			// get ip address for 1 ingress
+			klog.Info("Getting public IP from Ingress...")
+			publicIP, err := getPublicIP(clientset, namespaceName)
+			Expect(err).To(BeNil())
+			Expect(publicIP).ToNot(Equal(""))
+
+			url := fmt.Sprintf("https://%s/status/200", publicIP)
+
+			// simple hostname
+			_, err = makeGetRequest(url, "www.extended.com", 200, true)
+			Expect(err).To(BeNil())
+
+			// wilcard host name on multiple hostnames wildcard listener
+			_, err = makeGetRequest(url, "app.extended.com", 200, true)
```

Comment:
Should we also verify that request is going to the correct backend for each hostname ?
Right now, it only checks that 200 OK is recieved which can happen even when rule priority is not used.